update.sh

```diff
@@ -9,25 +9,58 @@ if [ ${#versions[@]} -eq 0 ]; then
 fi
 versions=( "${versions[@]%/}" )
 
-packagesBase='http://apt.postgresql.org/pub/repos/apt/dists/jessie-pgdg'
-mainList="$(curl -fsSL "$packagesBase/main/binary-amd64/Packages.bz2" | bunzip2)"
+declare -A debianSuite=(
+	[9.2]='jessie'
+	[9.3]='jessie'
+	[9.4]='jessie'
+	[9.5]='jessie'
+	[9.6]='jessie'
+	[10]='stretch'
+)
+declare -A alpineVersion=(
+	[9.2]='3.5'
+	[9.3]='3.5'
+	[9.4]='3.5'
+	[9.5]='3.5'
+	[9.6]='3.5'
+	[10]='3.6'
+)
 
 # https://www.mirrorservice.org/sites/ftp.ossp.org/pkg/lib/uuid/?C=M;O=D
 osspUuidVersion='1.6.2'
 osspUuidHash='11a615225baa5f8bb686824423f50e4427acd3f70d394765bdff32801f0fd5b0'
 
+lastSuite=
 travisEnv=
 for version in "${versions[@]}"; do
+	if [ "${debianSuite[$version]}" != "$lastSuite" ]; then
+		lastSuite="${debianSuite[$version]}"
+		packagesBase='http://apt.postgresql.org/pub/repos/apt/dists/'"${debianSuite[$version]}"'-pgdg'
+		mainList="$(curl -fsSL "$packagesBase/main/binary-amd64/Packages.bz2" | bunzip2)"
+	fi
```

Comment:
IMO since we don't run this very often, we might as well just ditch the optimization entirely and redownload every time (or use an associative array for cache instead).